koekalenteri-frontend/src/i18n/locales/en.json

```diff
@@ -0,0 +1,24 @@
+{
+  "dateformat": "MM/dd/yyyy",
```

Comment:
Ei näin. Jenkit on ainoa maa joka käyttää MM/dd/yyyy:tä käytännössä, se ei siis ole en-oletusformaatti. Moni Suomessa pitää selaimen englannilla. Oletusformaatti enkulla voisi olla vaikka yyyy-mm-dd tai sitten sama kuin fi/se kun kerran Euroopassa ollaan. Eli locale asetukset saa vaikka tulla maalta FI vaikka kieli vaihtuisi.